modes/tracker.md

```
@@ -1,23 +1,27 @@
-# Modo: tracker — Tracker de Aplicaciones
+# Mode: tracker — Applications Tracker
 
-Lee y muestra `data/applications.md`.
+Read and display `data/applications.md`.
 
-**Formato del tracker:**
+**Tracker Format:**
 ```markdown
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)